src/pages/page-names.constants.ts

```diff
@@ -56,7 +56,7 @@ export const CAT_C = {
   PASS_FINALISATION_PAGE: 'PassFinalisationCatCPage',
   REVERSE_DIAGRAM_PAGE: 'ReverseDiagramCatCPage',
   TEST_REPORT_PAGE: 'TestReportCatCPage',
-  WAITING_ROOM_PAGE: 'WaitingRoomCatCPage',
+  WAITING_ROOM_PAGE: 'WaitingRoomCatDPage',
```

Comment:
Undo this please as this is a Cat C Page

Reply:
done

Reply:
done